### ISSUE TITLE
[iOS Bootcamp] - Project 1 - Review my app

### ISSUE THREAD

**Issue by sarchak** [sarchak/RottenTomatoes#1]
My app is complete please review. /cc @codepath @thecodepath @codepathreview

**Comment by dirkdk** [comment]
great app.
- your segmented control should be visible for both box and DVD
- try to get rid of all warnings
- for an app like this, I would disable landscape mode and go portrait only. On my iPhone 5s, the table cells are too large

**Comment by sarchak** [comment]
Sure I will  modify the app. @dirkdk
 Should we be changing the cell size dynamically depending on what phone resolution it is?

**Comment by dirkdk** [comment]
yes, and as most iphones (5, 5s) have a with of 320 dots, make sure that works fine. 
http://www.paintcodeapp.com/news/iphone-6-screens-demystified

**Comment by dirkdk** [comment]
Oh, and you missed the segmented control to switch between list and icon view. I can see the code, but when I run the app I dont see the control, and can only use the app in list view

**Comment by sarchak** [comment]
@dirkdk 
No I have it in the  app https://github.com/sarchak/RottenTomatoes/raw/master/Rotten_Tomatoes.gif. Its in the other tab.
